backend/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:slim
+
+WORKDIR /app
+
+COPY package.json .
+
+COPY package-lock.json .
```

Comment:
the project was bootstrapped with `yarn` instead of npm and currently has `yarn.lock` files so probably need to replace it with npm then and use `package-lock` instead